_posts/2018-12-09-Welcome.md

```diff
@@ -0,0 +1,20 @@
+---
+author: Polybar Team
+account: polybar
+title: Welcome to our Blog!
```

Comment:
How should we capitalize our titles?
Also, is it possible to have automatically assign the account given the author? (@TonCherAmi)

Reply:
I generally go the route of capitalizing everyhing but unimportant words. [Grammar for Dummies](https://www.dummies.com/education/language-arts/grammar/what-do-you-capitalize-in-titles/) seems to suggest the same thing. Not sure if `our` should be uppercase or not.

>  Also, is it possible to have automatically assign the account given the author?

He used the following code to generate the github link:

```
https://github.com/{% if post.account %}{{ post.account }}{% else %}{{ post.author }}{% endif %}
```

So, if `account` is used as username, if it's set, otherwise `author` is used. So, yes, it's possible.

Reply:
If we're using [title case](https://en.wikipedia.org/wiki/Letter_case#Title_case) then the general rule seems to be to capitalize everything except for articles and short prepositions/conjunctions, meaning that 'our' should be capitalized.

Reply:
Fair enough. Changed it.